### DISCUSSION TITLE
Next.js adding unecssary empty comment <!-- --> when using SSR

### DISCUSSION THREAD

**Discussion by khanakia** [vercel/next.js#26350] [Help]
As you can see below code output it added `<!-- -->` this comment. how do i prevent this issue.

```
const Post = ({title}) => {
  return (
    <>
      <h3>Post {title}</h3>
    </>
  )
}

export const getServerSideProps = async (context) => {
  const slug = context.params.slug
  return {
    props: {
      title: slug,
    }
  }  

}
export default Post
```

Ouputs http://localhost:3000/product/test
```
<h3>Post <!-- -->test</h3>
```

<img width="1021" alt="Screenshot 2021-06-19 at 7 36 16 AM" src="https://user-images.githubusercontent.com/2174170/122628002-112f8000-d0d1-11eb-87b6-dd7384b7ec16.png">

**Comment by khanakia** [comment]
@willemliu, @fabb, @carlosbaraza, @lachlanjc anybody can help regarding above issue ?

**Reply by willemliufdmg** [reply]
This is a React-specific issue and it's by design:

https://github.com/facebook/react/issues/14725#issuecomment-458960400

**Reply by khanakia** [reply]
Thanks 👍
